configs/urls-config.ts

```diff
@@ -0,0 +1,3 @@
+export const URLS = {
+  GEOSTORE: 'https://production-api.globalforestwatch.org/v1/geostore/',
```

Comment:
Is this file really needed if its only used once in the code?

Reply:
@davidlstarr I will be adding endpoints here as I go to isolate endpoints to a single file

Reply:
Are there any other endpoints in the app atm? I honestly don't think this is needed and we can create this when there are multiple endpoints.